src/components/MartiniDeck/MartiniDeck.tsx

```diff
@@ -29,7 +29,7 @@ class MartiniDeck extends Component<{}, TypeState> {
     return this.state.drinkList.map((martini: Drink) => {
       //   console.log(martini);
       return (
-        <Link to={`/${martini.idDrink}`}>
+        <Link to={`/drinks/${martini.idDrink}`}>
```

Comment:
THE SOLUTION!!!!!!! Great thinking team to reach our for help when we were stuck. What a simple and effective solution. -Jordan